packages/backend-common/api-report.md

```diff
@@ -526,6 +526,22 @@ export type SearchResponseFile = {
   content(): Promise<Buffer>;
 };
 
+// Warning: (ae-missing-release-tag) "ServerTokenManager" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
It would be great if we could get rid of these warnings as well while we're at it - I tried to clean them up already elsewhere in this package.

Reply:
:+1: done in 40ea710c14